crates/db_views/src/local_image_view.rs

```diff
@@ -23,7 +24,7 @@ impl LocalImageView {
       .order_by(local_image::published.desc())
       .into_boxed();
 
-    if let Some(user_id) = user_id {
+    if let Some(user_id) = local_user_id {
       query = query.filter(local_image::local_user_id.eq(user_id))
```

Comment:
`Local_Images` use the local user ID instead of the person ID. IIRC a user's person ID and local user ID aren't necessarily the same. Should we use person ID for the `local_image` table instead of local user ID? It won't always be local users viewing their own media (admins can also do that), and not all users will be local to an instance. I presume admins should be able to review uploads from users on different instances as well.

Reply:
`local_image` can only contain data about images that were uploaded by local users, so it makes sense to use local user ID there, but there is also `remote_image`, which currently doesn't contain info about which user was responsible for uploading it.

But theoretically it might be quite a good feature indeed to add a person_id there (which we can get from post/comment authors) - it could for sure help with moderating remote users who are posting malicious images which end up in your server.

Reply:
Remote users can't upload to the `local_image` table, so you can really only moderate / take action on a local users images.

I could see this being useful tho, as long as its admin only.